build/emoji/make-emojis

```diff
@@ -140,6 +140,40 @@ File.open(File.join(project_root, 'utils', 'emoji.jsx'), 'w') do |f|
     puts "\nwrote #{emojis.length} emojis to utils/emoji.jsx"
 end
 
+webapp_server_dir = File.join(project_root, 'utils', 'server')
```

Comment:
Instead of creating the file in `utils/server/emoji_data.go`, maybe we could just create it as `emoji_data.go` at the root of the project? I don't think we need to bother creating a new folder for this that might get checked in automatically

Reply:
Good for me. Are you referring for `root of the project` to the `utils/` folder or the `project_root`

Reply:
Just the `project_root` folder